packages/analytics/lib/index.js

```diff
@@ -195,21 +195,41 @@ class FirebaseAnalyticsModule extends FirebaseModule {
   resetAnalyticsData() {
     return this.native.resetAnalyticsData();
   }
+  // TODO test nested items objects in structs file. not here
```

Comment:
Whats this?

Reply:
This was for checking the validity of `Item` objects in an array of `Items` which has been implemented: https://github.com/invertase/react-native-firebase/blob/afe213b9d9b2ea5185badb6f9c6f5b65d3e20873/packages/analytics/lib/structs.js#L33. Will remove.